botnim/vector_store/vector_store_es.py

```diff
@@ -38,6 +39,59 @@ def __init__(self, config, config_dir, production,
             logger.error(f"Failed to connect to Elasticsearch: {str(e)}")
             raise ConnectionError(f"Could not connect to Elasticsearch: {str(e)}")
 
+    def _build_search_query(self, query_text: str, embedding: List[float], 
```

Comment:
@akariv I was not sure wether I should move this method here or not. Would be glad to hear wdyt.